risc0/cargo-risczero/README.md

```diff
@@ -67,15 +69,31 @@ Use the `build-toolchain` command to build the toolchain locally for your host.
 
 ## build
 
-Use the `build` command to build a guest code for the target `riscv32im-risc0-zkvm-elf`. The resulting ELF binary will be saved on `target/riscv-guest/riscv32im-risc0-zkvm-elf/docker/`. Warning: this requires having docker installed.
+Use the `build` command to build guest code for the zkVM target `riscv32im-risc0-zkvm-elf` deterministically.
+
+The compiled ELF is saved in: `./target/riscv-guest/riscv32im-risc0-zkvm-elf/docker/`
+
+With this containerized build process, we ensure that all builds of your guest code, regardless of the machine or local environment, will produce the same ImageID. The ImageID, and its importance to [security,](https://dev.risczero.com/tech_faq#security) is explained in more detail in our [developer FAQ.](https://dev.risczero.com/tech_faq#zkvm-application-design)
+
+Note: The build command requires the docker CLI installed and in your PATH.
 
 ### Examples
```

Comment:
See other comment -- If we accept that suggestion as-is, we should make this singular.
```suggestion
### Example
```